framework_tvb/tvb/core/entities/model/model_datatype.py

```diff
@@ -113,6 +113,40 @@ class DataType(HasTraitsIndex):
     fk_from_operation = Column(Integer, ForeignKey('OPERATIONS.id', ondelete="CASCADE"))
     parent_operation = relationship(Operation, backref=backref("DATA_TYPES", order_by=id, cascade="all,delete"))
 
+    @property
+    def summary_info(self):
+        # type: () -> typing.Dict[str, str]
+
+        ret = {}
+        if self.title:
+            ret['Title'] = str(self.title)
+
+        columns = self.get_base_table_columns
+        for attribute in columns:
+            try:
+                attr_field = getattr(self, attribute)
+                attr_name = self.get_attribute_name(attribute)
+                if attr_name is not None:
+                    ret[attr_name] = str(attr_field)
+            except Exception:
+                pass
+        return ret
+
+    @property
+    def get_base_table_columns(self):
```

Comment:
if you use name `get_...` then  leave it a normal method and call with `self. get_base_table_columns()`
if you make it a `@property`, then a better name would have been `def base_table_columns(self):`
Particularly in this case I would use neither, but a private method (so not a property) `def _get_base_table_columns(self):`